config/SolrMarc/kule_index.properties

```
@@ -147,8 +152,16 @@ gmd_tesim = script(gmd.bsh), getGMD
 section_number_tesim = 245n
 section_name_tesim = 245p
 summary_holdings_tesim = 8563
-isbn_tesim = 020a, (pattern_map.isbn_clean)
+#isbn_tesim = 020a, (pattern_map.isbn_clean)
 issn_tesim = 022a, first
 uniform_title_tesim = 240a
 performers_tesim = 511a
 publisher_number_tesim = 028a
+
+# Holding information
+callnumber_tesi = 926a
+status_tesi = 926t
+access_title_tesi = 999m
+access_url_tesi = "https://www.ualberta.ca/kule-folklore-centre/"
+location_tesim = 926m
```

This is how symphony does this.  You'd also need to add `bmufa` and it's mapping to
https://github.com/ualbertalib/discovery/blob/master/config/SolrMarc/index_scripts/location_facet.bsh , https://github.com/ualbertalib/discovery/blob/master/config/locations.yml and https://github.com/ualbertalib/NEOSDiscovery/blob/master/config/locations.yml

```suggestion
location_tesim = script(location_facet.bsh), getLocationFacet
```